src/mpol/constants.py

```diff
@@ -1,11 +1,18 @@
+from __future__ import annotations
+
+from typing import Final
+
+import astropy.constants as apy_const
 import numpy as np
-from astropy.constants import c, k_B
 
 # convert from arcseconds to radians
-arcsec = np.pi / (180.0 * 3600)  # [radians]  = 1/206265 radian/arcsec
+arcsec: Final[float] = np.pi / (180.0 * 3600)  # [radians]  = 1/206265 radian/arcsec
+
+deg: Final[float] = np.pi / 180  # [radians]
 
-deg = np.pi / 180  # [radians]
+# Boltzmann constant
+kB: Final[float] = getattr(apy_const, "k_B").cgs.value  # [erg K^-1]
```

Comment:
What's the reason for writing this using `getattr` instead of just naming the constant? This seems more verbose and less intuitive.

Reply:
Is it necessary for the `Final` type hint (I'm assuming this means `const` or something like that)?

Reply:
mypy gives an error with the usual way because of the way astropy exports their constants. This is a temporary work around on my local machine that I forgot to change as I haven't gotten around to defining a mypy.ini file yet.

Reply:
Final will hint to mypy that the values shouldn't change. We can do away with it, though I would import constants.py as a module instead of importing the constants within it to signify to developers that the values shouldn't change. This way, it is always clear that, for instance, `const.kB` should not be altered.